EdFi.Buzz.Database/migrations/sqls/20201117102215-create-attendance-tables-up.sql

```diff
@@ -0,0 +1,17 @@
+-- SPDX-License-Identifier: Apache-2.0
+-- Licensed to the Ed-Fi Alliance under one or more agreements.
+-- The Ed-Fi Alliance licenses this file to you under the Apache License, Version 2.0.
+-- See the LICENSE and NOTICES files in the project root for more information.
+
+CREATE TABLE buzz.Attendance (
+    StudentSchoolKey VARCHAR(45) NOT NULL,
+    StudentKey VARCHAR(32) NOT NULL,
+    SchoolKey VARCHAR(30) NOT NULL,
```

Comment:
Do we need StudentKey and SchoolKey in this table? Maybe just StudentSchoolKey is fine.